release_notes/ocp_3_5_release_notes.adoc

```diff
@@ -0,0 +1,1683 @@
+introduces logic to the controllers that controls how many times they retry
+calls, and intelligently backs them off so that they do not hang.
+
+These controllers now contain the logic:
+
+* replication controller
+* replica set
+* daemonset
+* certificates
+* deployments
+* endpoints
+* pod disruption budget
+* jobs
+
+[[ocp-35-kubelet-collection-of-node-attriutes-for-scheduling-considerations]]
+==== Kubelet Collection of Node Attributes for Scheduling Considerations (Technology Preview)
+
+The kubelet is now able to collect any attribute on the node for scheduling
+considerations. This feature is currently in xref:ocp-35-technology-preview[Technology
+Preview].
+
+The cluster operator must advertise a per-node opaque resource on one or more
+nodes. Users must request the opaque resource in pods. To advertise a new opaque
+integer resource, the cluster operator should submit a PATCH HTTP request to the
+API server to specify the available quantity in the `status.capacity` for a node
+in the cluster. After this operation, the node's `status.capacity` will include
+a new resource. The `status.allocatable` field is updated automatically with the
+new resource asynchronously by the kubelet.
+
+[[ocp-35-statefulsets]]
+==== StatefulSets (Technology Preview)
+
+`StatefulSets` (currently in xref:ocp-35-technology-preview[Technology Preview]
+and formerly known as `PetSets`) offer more control over scale, network naming,
+handling of PVs, and deployment sequencing.
+
+This new controller allows for the deployment of applications types that require
+changes to their configuration or deployment count (instances) to be done in a
+specific and ordered manner.
+
+Supported:
+
+- Declaration of the Ordinal Index.
+- Stable network ID nomenclature.
+- Controlled or manual handling of PVs.
+- Sequence control at deployment time.
+- Ordered control during scale up or scale down, based on instance status.
+
+Not Supported:
+
+- Slow to iterate through the Ordinal Index and, therefore, slow on scale up and
+scale down.
+- No deployment or pod specification post deployment verification of what is
+deployed versus what is configured in the JSON file.
+- Locality awareness of zones or regions when dealing with scale up or scale down
+ordinality changes or mounted PVs.
+
+See more information about xref:ocp-35-web-console-statefulsets[web console
+enhancements] related to this feature for {product-title} 3.5.
+
+[[ocp-35-registry]]
+=== Registry
+
+{product-title} now allows control of whether or not an image is cached locally
+in the internal OpenShift Registry via the `oc tag` command with the
+`--reference-policy=local` and ``--scheduled=true` options.
+
+The storage of the manifest is moved to the OpenShift Registry, instead of
+storing it in etcd. There are two processes that will clean up existing images'
+metadata from etcd:
+
+* Gradually, `push` and `prune` will migrate all etcd images to not have manifest attached.
```

Comment:
@bfallonf "Gradually" should not be part of the list. I am trying to convey that the etcd images are being gradually migrated. I reworded this to add clarity.